performance-es.md

```diff
@@ -57,3 +57,117 @@ La optimización prematura también puede afectarte al atarte a ciertas decision
 En la gran mayoría de casos, el tamaño y velocidad del programa no es el problema. La optimización más fácil es no hacerla. La segunda alternativa más fácil es simplemente comprar mejor hardware.
 
 Cuando hayas decidido cambiar tu programa, sigue leyendo.
+
+### Modificar los datos
+
+Modificar los datos significa agregar o alterar la representación de los datos
+que estas procesando. Desde el punto de vista del rendimiento, algunos de
+estos acabaran cambiando la complejidad O() asociada a diferentes aspectos de
+la estructura de datos.
+
+Ideas para mejorar tu estructura de datos:
+
+* Campos adicionales
+
+  El clásico ejemplo de esto es almacenar la longitud de una lista enlazada en un
+  campo en el nodo raíz. Mantenerla actualizada lleva un poco más de trabajo,
```

Comment:
`s/lleva/toma`

Reply:
In Spain you would rather say `lleva` but `toma` also works. 

Alternatives that also work: `conlleva`, `supone`.

Reply:
I see, now that you mention that, either of them are fine 👍

Reply:
`conlleva` sounds good! changed it!